pkgs/gnomod/fetch.go

```diff
@@ -0,0 +1,26 @@
+package gnomod
+
+import (
+	"fmt"
+
+	abci "github.com/gnolang/gno/pkgs/bft/abci/types"
+	"github.com/gnolang/gno/pkgs/bft/rpc/client"
+)
+
+func QueryChain(qpath string, data []byte) (res *abci.ResponseQuery, err error) {
```

Comment:
```suggestion
func queryChain(qpath string, data []byte) (res *abci.ResponseQuery, err error) {
```

keep it unexported at the moment, so people don't use the gnomod package as a chain client.

Reply:
Resolved-by: aaccfe3